README.md

```diff
@@ -132,42 +132,23 @@ set on the event object to prevent API from rejecting an invalid request.
 
 ## Note On Server SSL Certificate Validation
 
-By default when validating DataSet endpoint server SSL certificate, logstash plugin will use a
-combination of system CA certs bundle from ``/etc/ssl/certs/ca-certificates.crt`` and combination
-of root CA certificates which are bundled with this plugin which represent root certificates used
-to issue / sign server certificates used by the DataSet API endpoint.
+By default when validating DataSet endpoint server SSL certificate, logstash uses CA certificate
+bundles which is vendored / bundled with the RubyGem / plugin. This bundle includes CA certificate
+files of authoried which are used to issue DataSet API endpoint certificates.
 
-In case you want to use only system CA certs bundle (not use certs which are bundled with the
-plugin), you can do that by using the following config options:
+If you want to use system CA bundle, you should update ``ssl_bundle_path`` to system CA bundle
```

Comment:
it seems the the `ca` is missed in config name.

Reply:
Thanks, fixed.